src/main/java/com/powsybl/openloadflow/lf/LoadFlowResult.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
Can I add the 'SPDX-License-Identifier: MPL-2.0` everywhere?

Reply:
Added